apps/apps-cli/src/cli.ts

```diff
@@ -0,0 +1,158 @@
+#!/usr/bin/env node
+
+import chalk from "chalk";
+import { createRequire } from "module";
+import semver from "semver";
+import yargs from "yargs";
+import { hideBin } from "yargs/helpers";
+import { installAppCommand } from "./commands/install-app-command";
+import { uninstallAppCommand } from "./commands/uninstall-app-command";
+import "dotenv/config";
```

Comment:
dotenv/config should be first line, beacuae 

```
import { installAppCommand } from "./commands/install-app-command";
import { uninstallAppCommand } from "./commands/uninstall-app-command";
```

these will not have access to .env